gen/azure/templates/acs.json

```diff
@@ -864,6 +864,7 @@
         "[variables('masterLbID')]",
         "[variables('vnetID')]",
         "[concat(variables('masterLbID'),'/inboundNatRules/SSH-',variables('masterVMNamePrefix'),copyIndex())]",
+        "[variables('masterSshPort22InboundNatRuleId')]",
```

Comment:
Good find... `masterSshPort22InboundNatRuleId` can't be created until the NIC on master0 is up, and the dependency on `masterLbID is not sufficient to ensure this (since the NAT rules themselves also depend on the LB being created).